synapse/tests/test_lib_stormsvc.py

```diff
@@ -697,3 +699,78 @@ async def test_storm_vars(self):
             self.stormIsInPrint('DEBUG: fooz=1.2.3.4', msgs)
             self.stormIsInPrint('my foo var is 5.6.7.8', msgs)
             self.stormIsInPrint('DEBUG: fooz=5.6.7.8', msgs)
+
+    async def test_storm_svc_mirror(self):
```

Comment:
This test was from https://github.com/vertexproject/synapse/tree/stormsvc-mirror-bug/synapse